<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="1" skipped="1">
  <reporter-output>
    <line>
      <![CDATA[TestNG_ReportsAndLogs -> This runs once before class]]>
    </line>
    <line>
      <![CDATA[TestNG_ReportsAndLogs -> This runs before every method]]>
    </line>
    <line>
      <![CDATA[TestNG_ReportsAndLogs -> testMethod2]]>
    </line>
    <line>
      <![CDATA[TestNG_ReportsAndLogs -> This runs after every method]]>
    </line>
    <line>
      <![CDATA[TestNG_ReportsAndLogs -> This runs before every method]]>
    </line>
    <line>
      <![CDATA[TestNG_ReportsAndLogs -> testMethod1]]>
    </line>
    <line>
      <![CDATA[TestNG_ReportsAndLogs -> This runs after every method]]>
    </line>
    <line>
      <![CDATA[TestNG_ReportsAndLogs -> This runs once after class]]>
    </line>
  </reporter-output>
  <suite started-at="2020-12-04T23:48:23 CET" name="Regression TestSuite" finished-at="2020-12-04T23:48:23 CET" duration-ms="85">
    <groups>
    </groups>
    <test started-at="2020-12-04T23:48:23 CET" name="Application Test" finished-at="2020-12-04T23:48:23 CET" duration-ms="85">
      <class name="testclasses.TestNG_ReportsAndLogs">
        <test-method is-config="true" signature="setUp()[pri:0, instance:testclasses.TestNG_ReportsAndLogs@78e94dcf]" started-at="2020-12-04T23:48:23 CET" name="setUp" finished-at="2020-12-04T23:48:23 CET" duration-ms="10" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[TestNG_ReportsAndLogs -> This runs once before class]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testclasses.TestNG_ReportsAndLogs@78e94dcf]" started-at="2020-12-04T23:48:23 CET" name="beforeMethod" finished-at="2020-12-04T23:48:23 CET" duration-ms="1" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[TestNG_ReportsAndLogs -> This runs before every method]]>
            </line>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="testMethod2()[pri:0, instance:testclasses.TestNG_ReportsAndLogs@78e94dcf]" started-at="2020-12-04T23:48:23 CET" name="testMethod2" finished-at="2020-12-04T23:48:23 CET" duration-ms="9" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at testclasses.TestNG_ReportsAndLogs.testMethod2(TestNG_ReportsAndLogs.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[TestNG_ReportsAndLogs -> testMethod2]]>
            </line>
          </reporter-output>
        </test-method> <!-- testMethod2 -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testclasses.TestNG_ReportsAndLogs@78e94dcf]" started-at="2020-12-04T23:48:23 CET" name="afterMethod" finished-at="2020-12-04T23:48:23 CET" duration-ms="1" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[TestNG_ReportsAndLogs -> This runs after every method]]>
            </line>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testclasses.TestNG_ReportsAndLogs@78e94dcf]" started-at="2020-12-04T23:48:23 CET" name="beforeMethod" finished-at="2020-12-04T23:48:23 CET" duration-ms="1" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[TestNG_ReportsAndLogs -> This runs before every method]]>
            </line>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="testMethod1()[pri:0, instance:testclasses.TestNG_ReportsAndLogs@78e94dcf]" started-at="2020-12-04T23:48:23 CET" name="testMethod1" finished-at="2020-12-04T23:48:23 CET" duration-ms="2" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[TestNG_ReportsAndLogs -> testMethod1]]>
            </line>
          </reporter-output>
        </test-method> <!-- testMethod1 -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testclasses.TestNG_ReportsAndLogs@78e94dcf]" started-at="2020-12-04T23:48:23 CET" name="afterMethod" finished-at="2020-12-04T23:48:23 CET" duration-ms="1" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[TestNG_ReportsAndLogs -> This runs after every method]]>
            </line>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method signature="testMethod3()[pri:0, instance:testclasses.TestNG_ReportsAndLogs@78e94dcf]" started-at="2020-12-04T23:48:23 CET" name="testMethod3" finished-at="2020-12-04T23:48:23 CET" depends-on-methods="testclasses.TestNG_ReportsAndLogs.testMethod2" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TestNG_ReportsAndLogs.testMethod3()[pri:0, instance:testclasses.TestNG_ReportsAndLogs@78e94dcf] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TestNG_ReportsAndLogs.testMethod3()[pri:0, instance:testclasses.TestNG_ReportsAndLogs@78e94dcf] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMethod3 -->
        <test-method is-config="true" signature="cleanUp()[pri:0, instance:testclasses.TestNG_ReportsAndLogs@78e94dcf]" started-at="2020-12-04T23:48:23 CET" name="cleanUp" finished-at="2020-12-04T23:48:23 CET" duration-ms="0" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[TestNG_ReportsAndLogs -> This runs once after class]]>
            </line>
          </reporter-output>
        </test-method> <!-- cleanUp -->
      </class> <!-- testclasses.TestNG_ReportsAndLogs -->
    </test> <!-- Application Test -->
  </suite> <!-- Regression TestSuite -->
</testng-results>
